dev/docs/assets/cupertino-icons.css

```diff
@@ -0,0 +1,29 @@
+/* <!-- Keep this file in sync with https://github.com/flutter/cupertino_icons/blob/master/css/icons.css --> */
+@font-face {
+  font-family: 'Cupertino Icons';
+  font-style: normal;
+  font-weight: 400;
+  src: url("https://flutter.github.io/cupertino_icons/assets/CupertinoIcons.ttf") format("truetype");
```

Comment:
should this be a local file? 

i.e. when we push cupertino_icons: 2.0.0 and the user's pubspec has ^1.0.0, wouldn't this show the wrong preview?

Reply:
It looks like the relative link in https://flutter.github.io/cupertino_icons/css/icons.css works. Removed